src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -0,0 +1,170 @@
+                                                                                                ModBlocks.blockHutCitizen, achievementGetSupply)
+            .registerStat();
+    /**
+     * Max out a builder
+     */
+    public static final Achievement achievementUpgradeColonistMax = new MineColoniesAchievement("upgrade.colonist.max", "upgrade.colonist.max", 2, 2,
+                                                                                                ModBlocks.blockHutCitizen, achievementBuildingColonist)
+            .registerStat();
+
+    /**
+     * Upgrade a lumberjack to lv 1
+     */
+    public static final Achievement achievementBuildingLumberjack   = new MineColoniesAchievement("upgrade.lumberjack.first", "upgrade.lumberjack.first", 0, 3,
+                                                                                                  ModBlocks.blockHutLumberjack, achievementGetSupply)
+            .registerStat();
+    /**
+     * Max out a lumberjack
+     */
+    public static final Achievement achievementUpgradeLumberjackMax = new MineColoniesAchievement("upgrade.lumberjack.max", "upgrade.lumberjack.max", 2, 3,
+                                                                                                  ModBlocks.blockHutLumberjack, achievementBuildingLumberjack)
+            .registerStat();
+
+    /**
+     * Upgrade a miner to lv 1
+     */
+    public static final Achievement achievementBuildingMiner   = new MineColoniesAchievement("upgrade.miner.first", "upgrade.miner.first", 0, 4,
+                                                                                             ModBlocks.blockHutMiner, achievementGetSupply)
+            .registerStat();
+    /**
+     * Max out a miner
+     */
+    public static final Achievement achievementUpgradeMinerMax = new MineColoniesAchievement("upgrade.miner.max", "upgrade.miner.max", 2, 4,
+                                                                                             ModBlocks.blockHutMiner, achievementBuildingMiner)
+            .registerStat();
+
+    /**
+     * Upgrade a fisher to lv 1
+     */
+    public static final Achievement achievementBuildingFisher   = new MineColoniesAchievement("upgrade.fisher.first", "upgrade.fisher.first", 0, 5,
+                                                                                              ModBlocks.blockHutFisherman, achievementGetSupply)
+            .registerStat();
+    /**
+     * Max out a fisher
+     */
+    public static final Achievement achievementUpgradeFisherMax = new MineColoniesAchievement("upgrade.fisher.max", "upgrade.fisher.max", 2, 5,
+                                                                                              ModBlocks.blockHutFisherman, achievementBuildingFisher)
+            .registerStat();
+
+    // Sizes
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)